pkg/service/test/test.go

```diff
@@ -465,17 +470,21 @@ func (t *tester) testHttp(tc models.TestCase, actualResponse *models.HttpResp) (
 	noise := tc.Noise
 
 	var (
-		bodyNoise   []string
-		headerNoise = map[string]string{}
+		bodyNoise   = noiseConfig["body"]
+		headerNoise = noiseConfig["header"]
 	)
 
 	for _, n := range noise {
 		a := strings.Split(n, ".")
 		if len(a) > 1 && a[0] == "body" {
 			x := strings.Join(a[1:], ".")
-			bodyNoise = append(bodyNoise, x)
+			if _, ok := bodyNoise[x]; !ok {
```

Comment:
noise written in test-case file should be given more preference than noise from config.

Reply:
Consider when body.ts is mentioned in noise field of a particular testcase yaml. 
Do you want a behaviour to ignore body.ts as a whole (equivalent of this in config would be `ts:[]`) ? 

Note: body.ts may also be defined in config file with just some regex values to ignore.

Reply:
Lets consider a case that even in testcase yaml we are supporting regex, Now I have mentioned body.ts = someregex in testcase yaml and in config also globally I have mentioned some other regex , So we want regex written in testcase yaml to be more prefered more which is not happening here.

Reply:
Do we want the regex array support in testcase yaml ?

Reply:
is this clear

Reply:
Yes.

Reply:
done!